DEVELOPER.md

```diff
@@ -0,0 +1,35 @@
+# logstash-output-zabbix
+
+## Testing Information
+
+### Unit Tests
+
+To run the unit tests:
+
+    bundle exec rspec -f d -c
+
+### Integration Tests
+
+This plugin has RSpec integration tests which use a Docker image.
+
+#### Pull the Docker image
+
+If you are using Docker already, you can prime your Docker instance with
+
+    docker pull untergeek/logstash_output_zabbix_rspec
```

Comment:
maybe make this more explicit like this:
```
docker pull untergeek/logstash_output_zabbix_rspec:zabbix_v2.2.2
```

my docker complained that it could not find an image with the "latest" tag (assuming it looks for that by default).